WORKSPACE

```diff
@@ -0,0 +1,2 @@
+# This file must be present for Bazel 6.
+# TODO(1.0) delete
```

Comment:
Should we add `WORKSPACE.bzlmod` as well to separate what workspace vs bzlmod runs? Then this WORKSPACE can `fail()`?